tests/qeqiskit/backend/backend_test.py

```diff
@@ -118,7 +124,7 @@ def test_aggregate_measurements(self, backend):
     def test_run_circuitset_and_measure(self, backend):
         # Given
         num_circuits = 10
-        circuit = self.x_cnot_circuit()
+        circuit = self.x_circuit()
```

Comment:
Why have you changed the circuit?

Reply:
Consistency. Not really any other reason. Is there a reason to keep it the way it was?

Reply:
No, it was just unclear to me.